src/core/exception.d

```diff
@@ -107,7 +107,7 @@ class ArrayIndexError : RangeError
         sink.rangeMsgPut("index [");
         sink.rangeMsgPut(unsignedToTempString!10(index, tmpBuf));
         sink.rangeMsgPut("]");
-        sink.rangeMsgPut(" exceeds array of length ");
+        sink.rangeMsgPut(" is out of bounds for array of length ");
```

Comment:
Thanks @nordlow ! Not about this PR, but seems like like 109 and 110 could be merged.

Reply:
Done at https://github.com/dlang/druntime/pull/3575.